src/main/java/codeu/model/data/Conversation.java

```diff
@@ -81,7 +85,10 @@ public void setMembers(List<UUID> members) {
   public List<UUID> getMembers() {
 	  return members;
   }
-
+  
+  public User getUserName(UUID uuid){
```

Comment:
This is the method I was thinking about. If we just put this same line inside ChatServlet then we don't need this method, or the userStore variable.